QModManager/OptionsManager.cs

```diff
@@ -60,7 +63,39 @@ internal static void Postfix(uGUI_OptionsPanel __instance)
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable debug logs", Config.EnableDebugLogs, new UnityAction<bool>(value => Config.EnableDebugLogs = value), null });
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable developer mode", Config.EnableDevMode, new UnityAction<bool>(value => Config.EnableDevMode = value), null });
                 }
+                #endregion Mod Config
+
+                #region Mod List
+                ModListTab = __instance.AddTab("QMods List");
+
+#if SUBNAUTICA_STABLE
+                __instance.AddHeading(ModListTab, $"QModManager Stable for Subnautica running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#elif SUBNAUTICA_EXP
+                __instance.AddHeading(ModListTab, $"QModManager Experimental for Subnautica running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#elif BELOWZERO_STABLE
+                __instance.AddHeading(ModListTab, $"QModManager Stable for Below Zero running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#elif BELOWZERO_EXP
+                __instance.AddHeading(ModListTab, $"QModManager Experimental for Below Zero running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#endif
+                
+                __instance.AddHeading(ModListTab, $"- - List of currently running Mods - -");
+
+                IEnumerable<IQMod> mods = QModServices.Main.GetAllMods().OrderBy(mod => mod.Id);
+                int modsactivecounter = 0;
+
+                foreach (var mod in mods)
+                {
+                    if(mod.Enable)
+                    {
+                        __instance.AddHeading(ModListTab, $"{mod.DisplayName} version {mod.ParsedVersion.ToString()} from {mod.Author}");
+                        modsactivecounter++;
+                    }
+                }
 
+                __instance.AddHeading(ModListTab, $"- - Statistics - -");
+                __instance.AddHeading(ModListTab, $"{modsactivecounter} Mods activated from {mods.Count()} loaded");
+                __instance.AddHeading(ModListTab, $"{mods.Count() - modsactivecounter} Mods inactive from {mods.Count()} total");
```

Comment:
I think I would like to see this ABOVE the mods list itself instead of below it.

Reply:
If its possible to archive that without going through the Modslist twice (refering to workload for people with bad setups) i am 100% on your side. It would look better.

Reply:
The amount of work required to go through the list twice in this case is negligible as it is not happening on update or anywhere that is called super often. 

Also the Orderby is ordering by id and your showing display name. I think the order should match what is displayed.

I was thinking something like this.

```cs

IEnumerable<IQMod> mods = QModServices.Main.GetAllMods().OrderBy(mod => mod.DisplayName);
List<IQMod> activeMods = new List<IQMod>();
List<IQMod> inactiveMods = new List<IQMod>();

foreach (var mod in mods)
{
    if(mod.Enable)
    {
        activeMods.Add(mod);
    }
    else
    {
        inactiveMods.Add(mod);
    }
}

__instance.AddHeading(ModListTab, $"- - Statistics - -");
__instance.AddHeading(ModListTab, $"{activeMods.Count} Mods activated from {mods.Count()} loaded");
__instance.AddHeading(ModListTab, $"{inactiveMods.Count} Mods inactive from {mods.Count()} total");

__instance.AddHeading(ModListTab, $"- - List of currently running Mods - -");
foreach (var mod in activeMods)
{
    __instance.AddHeading(ModListTab, $"{mod.DisplayName} version {mod.ParsedVersion.ToString()} from {mod.Author}");
}

__instance.AddHeading(ModListTab, $"- - List of Disabled Mods - -");

foreach(var mod in inactiveMods)
{
    __instance.AddHeading(ModListTab, $"{mod.DisplayName} version {mod.ParsedVersion.ToString()} from {mod.Author}");
}
```

Reply:
Integrated like requested in later Commit